website/content/docs/nia/configuration.mdx

```diff
@@ -113,15 +113,15 @@ service {
 }
 ```
 
-| Parameter |  Required |Description | Default |
-| --------- | --------- | ---------- | ------- |
-| `name` |  required | String value specifying the Consul logical name of the service. | none |
-| `id` |  Optional | String value specifying the service for CTS. This is used to explicitly identify the service config for a task to use. If no ID is provided, the service is identified by the service name within a [task definition](#task). | none |
-| `description` |  Optional | String value specifying the human readable text to describe the service | none |
-| `datacenter` |  Optional | String value specifying the name of a datacenter to query for the task. | Datacenter of the agent that CTS queries. |
-| `namespace`  | Optional | <EnterpriseAlert inline /> <br/> String value indicating the namespace of the services to query for the task. | In order of precedence: <br/> 1. Inferred from the CTS ACL token <br/> 2. The `default` namespace. |
-| `filter` |  Optional | String value specifying an expression used to additionally filter the services to monitor. <br/><br/> Refer to the [services filtering documentation](/api-docs/health#filtering-2) and section about [how to write filter expressions](/api-docs/features/filtering#creating-expressions) for additional information. | none |
-| `cts_user_defined_meta` | Optional | User-defined metadata specified as a map of strings that will be appended to the [service input variable](/docs/nia/terraform-modules#services-source-input) for compatible Terraform modules. By default, no metadata is appended. <br/><br/> Some modules do not use the configured metadata. Refer to the module configured for the task for information about metadata usage and expected keys and format. <br/><br/> If multiple tasks depend on the same service but require different metadata, you can declare different sets of metadata for the same service. Define multiple service blocks for the service with unique IDs (and identical names) for those blocks. The metadata can then be separated per task based on the service IDs. | none|
+| Parameter | Required | Type | Description | Default |
+| --------- | -------- | ---- | ----------- | ------- |
+| `name` |  Required | string | Consul logical name of the service. | none |
+| `id` |  Optional | String | Service ID for CTS. This is used to explicitly identify the service config for a task to use. If no ID is provided, the service is identified by the service name within a [task definition](#task). | none |
```

Comment:
small nit - looks like type here is capitalized "String" whereas other types are lower cased

Reply:
Good catch, thanks!